### PR TITLE
README: Instructions for running

### PR DESCRIPTION
**Pull request by nishantjr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# proof-generation-public
\ No newline at end of file
+# $\Pi^2$ Proof Checkers
+
+This repository contains implementations of the matchign logic proof checker using various ZK implementations.
```
Comment (nishantjr) [inline README.md]:
```suggestion
This repository contains implementations of the matchign logic proof checker using various ZK implementations.
Note that the code in this repository is experimental and exploratory.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+* Build:
+* Run without generating a certificate:
+* Generate a certificate:
+
+## Cairo
+
+In the `cairo` directory:
+* Build the checker: `scarb build`
+* Build and run the checker: `scarb cairo-run`
+* Run the tests: `scarb cairo-test`
```
Comment (nishantjr) [inline README.md]:
```suggestion
In the `cairo` directory:

* Build the checker: `scarb build`
* Build and run the checker: `scarb cairo-run`
* Run the tests: `scarb cairo-test`
```

File: README.md [hunk 3]
```diff
@@ -1 +1,30 @@
-# proof-generation-public
\ No newline at end of file
+# $\Pi^2$ Proof Checkers
```
Comment (xc93) [inline README.md]: Implementations of the Pi2 Proof Checker

File: README.md [hunk 4]
```diff
@@ -1 +1,30 @@
...
+
+This repository contains implementations of the matchign logic proof checker using various ZK implementations.
```
Comment (xc93) [inline README.md]: using various ZK implementations => in Cairo, Risc0, and zkLLVM.

Comment (xc93) [reply README.md]: Reorder the subsections accordingly (in alphabetical order).

**Review by xc93 — CHANGES_REQUESTED** [review]

**Review by xc93 — APPROVED** [review]

**Pull request merged**